### PR TITLE
Jeroen topic api

### PR DESCRIPTION
**Pull request by Jeroen777**
I've added an API to load unsplash images on a page.
I've  used a new package for this called request

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -192,6 +194,17 @@ app.get('/loginFailed', (req, res) => {
     
     };
 
+    // function to show the images from the unsplash API on the imagesGrid page
+    function showImages(req, res){
+      request("https://api.unsplash.com/photos?client_id=RsyiavOTOKJ65Sgr3PECQZ97VYTWbzAwEbHAVoR62Oc", function (error, response, body){
```
Comment (daoneandonly) [inline app.js]: Be careful with using private keys in the codebase. We can move this the `.env` file

**Review by daoneandonly — CHANGES_REQUESTED** [review]
Very nice! Be sure to be careful with private informations such as API keys.

**Comment by pepijnkok** [comment]
Looks good to me now!

**Review by daoneandonly — APPROVED** [review]
For a real app, we would use a check or give an alternative to the api key. For a prototype like we're making, this would be fine. Good work 👍

**Comment by Jeroen777** [comment]
I've merged the develop branch in to my API branch. Was a small problem with an url of my function, but it is fixed now.

**Pull request merged**